src/react/LoginCallBack.js

```diff
@@ -29,6 +29,8 @@ const LoginCallBack = ( { successCallback, errorCallback, customErrorComponent,
                     if ( authState.idToken )
                     {
                         updateAuthState( authState );
+                        setOCId(ocAuth.OCId)
+                        setEthAddress(ocAuth.ethAddress)
```

Comment:
so if `OCCOnnect` is changed like I suggested, then we don't need to set those states again here yet again.